Update android-installation.mdx (#73)
mirrorworld-universe/docs.mirrorworld.fun: data/android/android-installation.mdx

}
```

4. Config CustomTab!

> If you want to use CustomTab to show content to users(Recommand), you need to config the following contents on your AndroidManifest.xml:

Reviewer: ```suggestion
If you want to use CustomTab to show content to users (Recommended), you need to configure the following on your AndroidManifest.xml:
```

**Add permission of internet.**
```xml
<uses-permission android:name="android.permission.INTERNET" />
```

**Register this activity**
```xml
<activity
    android:name="com.mirror.sdk.activities.RedirectActivity"
    android:exported="true">

    <intent-filter>
        <action android:name="android.intent.action.VIEW"/>
        <category android:name="android.intent.category.DEFAULT"/>
        <category android:name="android.intent.category.BROWSABLE"/>
        <data android:scheme="mwsdk"/>
    </intent-filter>
</activity>
```

> So, the finnaly file would looked as this:

Reviewer: ```suggestion
The final file would look like this:
```

        android:label="@string/app_name"
        android:networkSecurityConfig="@xml/network_security_config"

        android:roundIcon="@mipmap/ic_launcher_round"
        android:supportsRtl="true"
        android:theme="@style/Theme.Mirrorworldsdkandroid"
        tools:targetApi="31">
        <activity
            android:name=".MainActivity"
            android:launchMode="singleTask"
            android:exported="true">
            <intent-filter>
                <action android:name="android.intent.action.MAIN" />
                <category android:name="android.intent.category.LAUNCHER" />
            </intent-filter>
        </activity>

        <activity
            android:name="com.mirror.sdk.activities.RedirectActivity"
            android:exported="true">

            <intent-filter>
                <action android:name="android.intent.action.VIEW"/>
                <category android:name="android.intent.category.DEFAULT"/>
                <category android:name="android.intent.category.BROWSABLE"/>
                <data android:scheme="mwsdk"/>
            </intent-filter>
        </activity>
    </application>

</manifest>
```

> **Add dependence**

Reviewer: ```suggestion
**Add dependency**
```

```xml
implementation 'androidx.browser:browser:1.4.0'
```

> And then, you will see CustomTab when you try to open our prepared web page.

Reviewer: ```suggestion
You should now see CustomTab when you try to open our prepared web page.
```

> >*Tips:If you want to make your app has only one task, you can set your own activity to SingTast:*

Reviewer: ```suggestion
>*Tips: If you want your app to have only one task, you can set your own activity to SingTast:*
```

String apiKey = "your-api-key";
boolean useDebugMode = true;

MirrorWorld.setDebug(useDebugMode);
MirrorWorld.initMirrorWorld(activity,apiKey,MirrorEnv.DevNet);
```

### Guide user to login
Users need to log in to use some MirrorSDK features (or APIs).

> If you want them to login(or again), you can use the following code:

Reviewer: ```suggestion
If you want them to login for the first time or once again, use the following code:
```

> Users may want to check their wallet in the app, you can use openWallet API:

Reviewer: ```suggestion
Users may want to check their wallet in the app, you can use the `openWallet` API for this:
```
